insta/src/runtime.rs

```diff
@@ -626,7 +660,7 @@ where
 #[allow(clippy::too_many_arguments)]
 pub fn assert_snapshot(
     refval: ReferenceValue<'_>,
```

Comment:
OoI, what is `refval` in the case of a binary snapshot? Presumably it won't be UTF-8, so the existing implementation won't always work?

Reply:
It should always be `ReferenceValue::Named` unless I missed something. Inline binary snapshots should not be possible. At least when going through the macros.

But it might be a problem if someone were to call it directly with a `ReferenceValue::Inline`. Although I think the function is only exposed throught `_macro_support` so I don't think that is really a problem. But maybe an assert would make sense?

Reply:
> It should always be `ReferenceValue::Named` unless I missed something. Inline binary snapshots should not be possible. At least when going through the macros.

I agree re `Named`, but that's `Named(Option<Cow<'a, str>>)`. IIUC that means the value needs to be a `str` — which in rust is a UTF-8 value.

Maybe I'm missing something — could we maybe add a test with a tiny png or other non-UTF-8 value to dispel this concern?

Reply:
Ah, sorry @lasernoises  — the value in `Named` is actually the snapshot name, not the value itself. My mistake.